index.html

```diff
@@ -14,10 +14,18 @@
         </div>
         <a class="menuim" href="#"><img src="./img/menu.svg" alt="Menu"></a>
         <ul class="nav_ul">
-            <li>Portfolio</li>
-            <li>About</li>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+        <div class="nav_expand" id="hide">
+            <ul class="expand_ims">   
+            <li><img class="iconx" src="./img/x.svg" alt="icon"></li>
+            <li id="ims">About</li>
             <li>Contact</li>
+            <li>Portfolio</li>
```

Comment:
- [ ] Kindly use the font family provided by the Figma template for the mobile menu buttons.
- [ ] When the user clicks on the menu buttons they should display the correct part of the page and close the mobile menu. Kindly fix that.
![image](https://user-images.githubusercontent.com/61919617/174820940-ae6ce779-fd3d-45e8-bfc1-32284cd0cfe0.png)